packages/appium/docs/base-mkdocs.yml

```diff
@@ -10,17 +10,20 @@ theme:
   favicon: assets/images/appium-logo.png
 
 plugins:
-  redirects:
-    redirect_maps:
-      'intro/requirements.md': 'quickstart/requirements.md'
-      'ecosystem/build-drivers.md': 'developing/build-drivers.md'
-      'ecosystem/build-plugins.md': 'developing/build-plugins.md'
-      'ecosystem/build-docs.md': 'developing/build-docs.md'
-      'reference/commands/base-driver.md': 'commands/base-driver.md'
-      'reference/commands/execute-driver-plugin.md': 'commands/execute-driver-plugin.md'
-      'reference/commands/images-plugin.md': 'commands/images-plugin.md'
-      'reference/commands/relaxed-caps-plugin.md': 'commands/relaxed-caps-plugin.md'
-      'reference/commands/universal-xml-plugin.md': 'commands/universal-xml-plugin.md'
+  - blog:
+      enabled: true
+      post_excerpt: required
+  - redirects:
```

Comment:
Almost missed this - this should not use the list syntax, otherwise no plugins can be inherited by e.g. Inspector/XCUITest docs.
https://www.mkdocs.org/user-guide/configuration/#configuration-inheritance

Reply:
ah good catch, fixed